models/lido/liquidity/ethereum/lido_liquidity_ethereum_balancer_pools.sql

```diff
@@ -0,0 +1,446 @@
+{{ config(
+    alias = 'balancer_pools',
+    partition_by = ['time'],
+    materialized = 'table',
+    file_format = 'delta',
+    unique_key = ['pool', 'time'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "lido_liquidity",
+                                \'["ppclunghe"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2021-08-13' %} 
+
+
+
+with dates as (
+select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 day)) as day
+)
+
+, selected_pools as (
+values       lower('0x32296969EF14EB0C6D29669C550D4A0449130230000200000000000000000080'), 
+              lower('0x5AEE1E99FE86960377DE9F88689616916D5DCABE000000000000000000000467'),
+              lower('0x9C6D47FF73E0F5E51BE5FD53236E3F595C5793F200020000000000000000042C'),
+              lower('0xE0FCBF4D98F0AD982DB260F86CF28B49845403C5000000000000000000000504'),
+              lower('0x5F1F4E50BA51D723F12385A8A9606AFC3A0555F5000200000000000000000465'),
+              lower('0x25ACCB7943FD73DDA5E23BA6329085A3C24BFB6A000200000000000000000387')
+              as pool_id
+)
+
+, pools as (   
+SELECT
+    registered.poolId AS pool_id,
+    evt_block_time
+FROM {{source('balancer_v2_ethereum','Vault_evt_PoolRegistered')}}  registered
+INNER JOIN {{source('balancer_v2_ethereum','WeightedPoolV2Factory_call_create')}} call_create
+    ON call_create.output_0 = SUBSTRING(registered.poolId, 0, 42)
+WHERE poolId in (select * from selected_pools)
```

Comment:
another note for future migration consideration:
you may get better performance on dunesql to do an inner join to `selected_pools` CTE here rather than an `in` statement

Reply:
of course, this would apply universally to any instance